src/dynamics/ergodic/ergodic.lean

```diff
@@ -68,6 +68,45 @@ lemma of_iterate (n : ℕ) (hf : pre_ergodic (f^[n]) μ) : pre_ergodic f μ :=
 
 end pre_ergodic
 
+namespace measure_theory.measure_preserving
+
+variables {β : Type*} {m' : measurable_space β} {μ' : measure β} {s' : set β} {g : α → β}
+
+lemma pre_ergodic_of_pre_ergodic (hg : measure_preserving g μ μ')
+  (hf : pre_ergodic f μ) {f' : β → β} (h_comm : g ∘ f = f' ∘ g) :
+  pre_ergodic f' μ' :=
+⟨begin
+  intros s hs₀ hs₁,
+  replace hs₁ : f⁻¹' (g⁻¹' s) = g⁻¹' s, { rw [← preimage_comp, h_comm, preimage_comp, hs₁], },
+  cases hf.ae_empty_or_univ (hg.measurable hs₀) hs₁ with hs₂ hs₂;
+  [left, right],
+  { simpa only [ae_eq_empty, hg.measure_preimage hs₀] using hs₂, },
+  { simpa only [ae_eq_univ, ← preimage_compl, hg.measure_preimage hs₀.compl] using hs₂, },
+end⟩
+
+lemma pre_ergodic_iff_pre_ergodic {e : α ≃ᵐ β} (h : measure_preserving e μ μ') :
```

Comment:
```suggestion
lemma pre_ergodic_iff_pre_ergodic_conjugate {e : α ≃ᵐ β} (h : measure_preserving e μ μ') :
```
And maybe you should switch sides in the iff, to put the more complicated version on the left. Then the name would become `pre_ergodic_conjugate_iff`.

Reply:
Good point; done.